test/test-2.0.rb

```diff
@@ -173,7 +173,7 @@ def test_channel_image
       image_params.each do |name, value|
         value = image.__send__(name)
         actual = image_elem.elements[name.to_s].text
-        actual = actual.to_i if [:width, :height].include?(name)
+        actual = actual.to_i if %i[width height].include?(name)
```

Comment:
```suggestion
        actual = actual.to_i if [:width, :height].include?(name)
```

Not in `Style/StringLiterals`.